### PR TITLE
Crm tests

### PR DESCRIPTION
**Pull request by andriymoroz-mlnx**
Signed-off-by: Volodymyr Samotiy <volodymyrs@mellanox.com>

Initial CRM test implementation for review. Please do not merge yet.

recreated https://github.com/lguohan/sonic-swss/pull/3 against Azure/master
depends on https://github.com/Azure/sonic-swss/pull/447 + submodule update in buildimage

### CODE REVIEW

**Comment by prsunny** [comment]
Please resolve conflict!

The previous review comments for this PR is at
https://github.com/lguohan/sonic-swss/pull/3#pullrequestreview-93719557

**Comment by prsunny** [comment]
Looks like few of the test cases failed. Can you please check?

**Comment by lguohan** [comment]
we have merged your CRM PR, not sure why some of the CRM test still fails, can you check?

**Comment by volodymyrsamotiy** [comment]
OK, thanks. Yes, I am working on it, already fixed almost all tests (just neighbor test is left).
I will update this PR once I am done.

**Comment by prsunny** [comment]
Few of the IPv6 test cases are failing.

**Comment by prsunny** [comment]
Can you click on the `"vs"` details above? It will take you to below page and you can check the `console output
`
https://sonic-jenkins.westus2.cloudapp.azure.com/job/vs/job/sonic-swss-build-pr/485/

**Comment by lguohan** [comment]
retest this please

File: tests/test_crm.py [hunk 1]
```diff
@@ -0,0 +1,517 @@
...
+    ntf.send("set_ro", key, fvp)
+
+
+def test_CrmFdbEntry(dvs):
+
+    dvs.runcmd("crm config polling interval 1")
+
+    setReadOnlyAttr(dvs, 'SAI_OBJECT_TYPE_SWITCH', 'SAI_SWITCH_ATTR_AVAILABLE_FDB_ENTRY', '1000')
+
+    time.sleep(5*60)
```
Comment (lguohan) [inline tests/test_crm.py]: do we still need to wait for 300 minutes here? I saw you change the polling interval to 1 second now.

File: tests/test_crm.py [hunk 2]
```diff
@@ -0,0 +1,517 @@
...
+    setReadOnlyAttr(dvs, 'SAI_OBJECT_TYPE_SWITCH', 'SAI_SWITCH_ATTR_AVAILABLE_IPV4_ROUTE_ENTRY', '1000')
+
+    # add static neighbor
+    dvs.runcmd("ip neigh replace 10.0.0.1 lladdr 11:22:33:44:55:66 dev Ethernet0")
+
+    db = swsscommon.DBConnector(0, dvs.redis_sock, 0)
+    ps = swsscommon.ProducerStateTable(db, "ROUTE_TABLE")
+    fvs = swsscommon.FieldValuePairs([("nexthop","10.0.0.1"), ("ifname", "Ethernet0")])
+
+    time.sleep(5*60)
```
Comment (lguohan) [inline tests/test_crm.py]: I believe the crm config polling interval take effort immediately, do we still need to wait for 300 seconds?

File: tests/test_crm.py (hunk 1 above)
Comment (prsunny) [reply tests/test_crm.py]: @lguohan, first time setting the polling interval will take effect only after the previous polling interval. Future wait would be _1 sec_

**Comment by volodymyrsamotiy** [comment]
Fixed issues with IPv6 test cases.

**Comment by prsunny** [comment]
Can you please resolve the conflicts with master branch?

**Pull request merged**